src/patterns/OcAutocomplete.vue

```diff
@@ -7,33 +7,37 @@
       :disabled="disabled"
       @keydown.up.prevent="highlighted--"
       @keydown.down.prevent="highlighted++"
-      @keydown.enter="selectSuggestion"
+      @keydown.enter="$_ocAutocomplete_selectSuggestion"
     />
-    <div hidden :id="_boundryId" />
+    <div hidden :id="$_ocAutocomplete_boundryId" />
     <div
       class="oc-autocomplete-dropdown"
-      :uk-drop="'mode:click;delay-hide:0;toggle:#' + _boundryId"
-      :id="_dropdownId"
+      :uk-drop="'mode:click;delay-hide:0;toggle:#' + $_ocAutocomplete_boundryId"
+      :id="$_ocAutocomplete_dropdownId"
     >
       <ul class="oc-autocomplete-suggestion-list">
-        <template v-for="(item, i) in matchesShown">
+        <template v-for="(item, i) in $_ocAutocomplete_matchesShown">
           <li
             :class="[
               'oc-autocomplete-suggestion',
               { 'oc-autocomplete-suggestion-selected': i === highlighted },
             ]"
             :key="i"
-            v-text="item"
             @mouseenter="highlighted = i"
-            @click="selectSuggestion"
-          ></li>
+            @click="$_ocAutocomplete_selectSuggestion"
+          >
+            <slot name="item" v-bind:item="item">
+              <!-- Fallback content -->
+              {{ item }}
+            </slot>
+          </li>
         </template>
         <li
-          v-if="matchesOverflowing > 0 && !itemsLoading"
+          v-if="$_ocAutocomplete_matchesOverflowing > 0 && !itemsLoading"
           class="oc-autocomplete-suggestion-overflow"
         >
-          {{ matchesOverflowing }}
-          <span v-if="matchesOverflowing === 1">more result</span>
+          {{ $_ocAutocomplete_matchesOverflowing }}
+          <span v-if="$_ocAutocomplete_matchesOverflowing === 1">more result</span>
```

Comment:
we need a property where we can set this user facing string from outside - because of translations ...

Reply:
We could have one `text` property that expects an object like:

`{ loading : 'Waiting for results', xMoreResults : '%s more results', oneMoreResult : '1 more result' }`